File: lumina_beauty_final_version/lumina_beauty_admin_frontend/SoluLumina/Models/Producto.cs
using Newtonsoft.Json;

namespace SoluLumina.Models
{
    public class Producto
    {
        private int id_producto;
        private String nombre;
        private String sku;
        private String slug;
        private String descripcion;
        private Decimal precio;
        private int stock;
        private String tipo_piel;
        private String imagen_url;
        private int estado;
        private DateTime fecha_creacion;
        private DateTime fecha_actualizacion;

        private CategoriaProducto categoria;
        private Marca marca;

        [JsonProperty("id_producto")]
        public int Id_producto { get => id_producto; set => id_producto = value; }
        [JsonProperty("nombre")]
        public string Nombre { get => nombre; set => nombre = value; }
        [JsonProperty("sku")]
        public string Sku { get => sku; set => sku = value; }
        [JsonProperty("slug")]
        public string Slug { get => slug; set => slug = value; }
        [JsonProperty("descripcion")]
        public string Descripcion { get => descripcion; set => descripcion = value; }
        [JsonProperty("precio")]
        public decimal Precio { get => precio; set => precio = value; }
        [JsonProperty("stock")]
        public int Stock { get => stock; set => stock = value; }
        [JsonProperty("tipoPiel")]
        public string Tipo_piel { get => tipo_piel; set => tipo_piel = value; }
        [JsonProperty("imagen_url")]
        public string Imagen_url { get => imagen_url; set => imagen_url = value; }
        [JsonProperty("estado")]
        public int Estado { get => estado; set => estado = value; }
        [JsonProperty("fecha_creacion")]
        public DateTime Fecha_creacion { get => fecha_creacion; set => fecha_creacion = value; }
        [JsonProperty("fecha_actualizacion")]
        public DateTime Fecha_actualizacion { get => fecha_actualizacion; set => fecha_actualizacion = value; }
        [JsonProperty("categoria")]
        public CategoriaProducto Categoria { get => categoria; set => categoria = value; }
        [JsonProperty("marca")]
        public Marca Marca { get => marca; set => marca = value; }
    }
}

File: lumina_beauty_final_version/lumina_beauty_backend/LuminaBeauty-Model/src/main/java/pe/edu/pucp/luminaBeauty/Model/Producto.java
package pe.edu.pucp.luminaBeauty.Model;

import java.math.BigDecimal;
import java.time.LocalDateTime;

public class Producto {

    private int id_producto;
    private String nombre;
    private String sku;
    private String slug;
    private String descripcion;
    private BigDecimal precio;
    private int stock;
    private String tipo_piel;
    private String imagen_url;
    private int estado;
    private LocalDateTime fecha_creacion;
    private LocalDateTime fecha_actualizacion;

    private CategoriaProducto categoria;
    private Marca marca;

    public Producto() {
        this.estado = 1;
        this.stock = 0;
    }

    public int getId_producto() {
        return id_producto;
    }

    public void setId_producto(int id_producto) {
        this.id_producto = id_producto;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getSku() {
        return sku;
    }

    public void setSku(String sku) {
        this.sku = sku;
    }

    public String getSlug() {
        return slug;
    }

    public void setSlug(String slug) {
        this.slug = slug;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public BigDecimal getPrecio() {
        return precio;
    }

    public void setPrecio(BigDecimal precio) {
        this.precio = precio;
    }

    public int getStock() {
        return stock;
    }

    public void setStock(int stock) {
        this.stock = stock;
    }

    public String getTipoPiel() {
        return tipo_piel;
    }

    public void setTipoPiel(String tipo_piel) {
        this.tipo_piel = tipo_piel;
    }

    public String getImagenUrl() {
        return imagen_url;
    }

    public void setImagenUrl(String imagen_url) {
        this.imagen_url = imagen_url;
    }

    public int getEstado() {
        return estado;
    }

    public void setEstado(int estado) {
        this.estado = estado;
    }

    public LocalDateTime getFecha_creacion() {
        return fecha_creacion;
    }

    public void setFecha_creacion(LocalDateTime fecha_creacion) {
        this.fecha_creacion = fecha_creacion;
    }

    public LocalDateTime getFecha_actualizacion() {
        return fecha_actualizacion;
    }

    public void setFecha_actualizacion(LocalDateTime fecha_actualizacion) {
        this.fecha_actualizacion = fecha_actualizacion;
    }

    public CategoriaProducto getCategoria() {
        return categoria;
    }

    public void setCategoria(CategoriaProducto categoria) {
        this.categoria = categoria;
    }

    public Marca getMarca() {
        return marca;
    }

    public void setMarca(Marca marca) {
        this.marca = marca;
    }
}